engine/runtime/src/test/java/org/enso/interpreter/test/VectorTest.java

```diff
@@ -133,7 +133,7 @@ public void passingListOrArrayToEnsoAsArray() throws Exception {
     import Standard.Base.Data.Array
 
     how_long array = case array of
-        Array.Array -> array.length
+        arr -> arr.length
```

Comment:
Something looks off here - the fallback `_ -> -1` becomes unreachable.

Shouldn't this be translated to the following?

```suggestion
        _: Array -> arr.length
```

Reply:
Good catch!